runtime/integration-tests/src/xcm/xcm_transfers.rs

```diff
@@ -410,36 +253,25 @@ fn currency_id_convert_air() {
 
 // The fee associated with transferring Native tokens
 fn native_fee() -> Balance {
-	let (_asset, fee) = NativePerSecond::get();
-	// We divide the fee to align its unit and multiply by 4 as that seems to be the unit of
-	// time the transfers take.
-	// NOTE: it is possible that in different machines this value may differ. We shall see.
-	fee.div_euclid(10_000) * 4
-}
-
-// The fee associated with transferring Native tokens
-fn usd_fee() -> Balance {
-	let (_asset, fee) = UsdPerSecond::get();
+	let (_asset, fee) = AirPerSecond::get();
 	// We divide the fee to align its unit and multiply by 4 as that seems to be the unit of
 	// time the transfers take.
 	// NOTE: it is possible that in different machines this value may differ. We shall see.
-	fee.div_euclid(10_000) * 4
+	fee.div_euclid(10_000) * 8
```

Comment:
This value doubled after the doubling of `UnitWeightCost` (was `100m` in development, now `200m` in altair)